variable/reduction.cpp

```diff
@@ -15,6 +15,8 @@
 #include "scipp/variable/special_values.h"
 #include "scipp/variable/transform.h"
 
+#include "operations_common.h"
```

Comment:
Has this been included by error? Or for another reason?

Reply:
Without this we get linker errors on certain platforms. This is the header declaring things such as `sum_impl` which gets defined in this file. Apparently the DllExport macros cause these to be disconnected somehow if the source file does not include respective includes.